lib/page/sign_up.dart

```diff
@@ -0,0 +1,175 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:toktik/controller/user_controller.dart';
+import 'package:toktik/res/colors.dart';
+import 'package:get/get.dart';
+
+class SignUpPage extends StatefulWidget {
```

Comment:
The class name is inconsistent with the file name. You should name the file as `sign_up_page.dart`